internal/config/config.go

```diff
@@ -29,6 +29,11 @@ type Config struct {
 		Namespace struct {
 			Enabled bool `envconfig:"default=true"`
 		}
+		Creator struct {
+			Enabled       bool   `envconfig:"default=true"`
+			RootNamespace string `envconfig:"default=openmfp-system"`
```

Comment:
I think we should use 
https://github.com/openmfp/helm-charts/blob/main/charts/example-resources/values.yaml#L3 as a default 
so `openmfp-root` since openmfp-system is used for our pods, not for the custom resources, that are getting created, when running openmfp